src/main/resources/db.scripts.mvp/R__VB-819_add_hewell_code_HEI_prison_data.sql

```diff
@@ -0,0 +1,40 @@
+-- MVP data for Hewell visit schedule
```

Comment:
I take it this folder holds manual scripts separate from the migration directory

Reply:
Yes, that's correct we want this script to be run manually!